nexus/src/app/background/tasks/region_replacement.rs

```diff
@@ -101,12 +98,14 @@ impl BackgroundTask for RegionReplacementDetector {
                     Ok(v) => v,
 
                     Err(e) => {
-                        error!(
-                            &log,
+                        let s = format!(
                             "error looking for existing region replacement \
                              requests for {}: {e}",
                             region.id(),
                         );
+                        error!(&log, "{s}");
```

Comment:
similarly, it might be nice to make the IDs be structured fields in the log entry and only interpolate them into the error message when formatting the string for the status? not a big deal.

Reply:
(similar feedback applies to the other log messages here, of course)

Reply:
see above, going to skip this suggestion